sdk/react/src/lib/fragment-components/a-time-picker.tsx

```diff
@@ -0,0 +1,379 @@
+			labelText='Placeholder'
+			onChange={(event: any) => setComponent({
+				...selectedComponent,
+				placeholder: event.currentTarget.value
+		})} />
+		<DraggableTileList
+			dataList={[...selectedComponent.items]}
+			setDataList={updateStepList}
+			updateItem={updateListItems}
+			defaultObject={{
+				text: 'New timezone',
+				value: 'New timezoneValue'
+			}}
+			template={template} />
+	</>;
+};
+
+export const ATimePickerCodeUI = ({ selectedComponent, setComponent }: any) => <TextInput
+	value={selectedComponent.codeContext?.name}
+	labelText='Input name'
+	onChange={(event: any) => setComponent({
+		...selectedComponent,
+		codeContext: {
+			...selectedComponent.codeContext,
+			name: event.currentTarget.value
+		}
+	})}
+/>;
+
+export const ATimePicker = ({
+	componentObj,
+	...rest
+}: any) => {
+	return (
+		<AComponent
+		componentObj={componentObj}
+		rejectDrop={true}
+		{...rest}>
+			{
+				<TimePicker
+				className={cx(
+					preventCheckEventStyle,
+					componentObj.cssClasses?.map((cc: any) => cc.id).join(' '),
+					css`${styleObjectToString(componentObj.style)}`
+				)}
+				disabled={componentObj.disabled}
+				invalid={componentObj.invalid}
+				invalidText={componentObj.invalidText}
+				placeholder={componentObj.placeholder}
+				hideLabel={componentObj.hideLabel}
+				labelText={componentObj.label}
+				light={componentObj.light}
+				size={componentObj.size}>
+					<TimePickerSelect disabled={componentObj.disabled} labelText='time-picker-1'>
+						<SelectItem value='AM' text='AM' />
+						<SelectItem value='PM' text='PM' />
+					</TimePickerSelect>
+					<TimePickerSelect disabled={componentObj.disabled} labelText='time-picker-2'>
+						{
+							componentObj.items.map((step: any, index: number) => <SelectItem
+								value={step.value}
+								text={step.text}
+								key={index}
+							/>)
+						}
+					</TimePickerSelect>
+				</TimePicker>
+			}
+		</AComponent>
+	);
+};
+
+export const componentInfo: ComponentInfo = {
+	component: ATimePicker,
+	settingsUI: ATimePickerSettingsUI,
+	codeUI: ATimePickerCodeUI,
+	keywords: ['timepicker', 'time', 'picker'],
+	name: 'Time Picker',
+	type: 'time-picker',
+	defaultComponentObj: {
+		type: 'time-picker',
+		disabled: false,
+		invalid: false,
+		invalidText: 'A valid value is required',
+		placeholder: 'hh:mm',
+		label: 'Select a time',
+		hideLabel: false,
+		light: false,
+		size: 'md',
+		value: '',
+		items: [
+			{
+				value: 'timezoneValue1',
+				text: 'Time zone 1'
+			},
+			{
+				value: 'timezoneValue2',
+				text: 'Time zone 2'
+			}
+		]
+	},
+	image,
+	codeExport: {
+		angular: {
+			latest: {
+                inputs: ({ json }) => `@Input() ${nameStringToVariableString(json.codeContext?.name)}Value = "${json.value};
+					@Input() ${nameStringToVariableString(json.codeContext?.name)}Theme = "${json.light ? 'light' : 'dark'}";
+					@Input() ${nameStringToVariableString(json.codeContext?.name)}IsInvalid = "${json.invalid}";
```

Comment:
Should this be a string? `invalid` should be a boolean.

```suggestion
					@Input() ${nameStringToVariableString(json.codeContext?.name)}IsInvalid = ${json.invalid};
```